src/tracer.js

```diff
@@ -292,7 +298,13 @@ export default class Tracer {
             throw new Error(`Unsupported format: ${format}`);
         }
 
-        return extractor.extract(carrier);
+        let spanContext = extractor.extract(carrier);
+
+        // spanContext can be null if using a differnt type of extractor. i.e. BinaryCodec
```

Comment:
irrelevant comment. Span context can be null purely because there was no context in the carrier.